src/styles/index.scss

```diff
@@ -1,3 +1,152 @@
-body {
+.body {
   margin: 0;
 }
```

Comment:
Block `.body` should be named `.page` or `.page__body` to follow BEM methodology as mentioned in the checklist. It's recommended to use the 'page' block for general styling.